core/src/main/java/org/apache/accumulo/core/fate/zookeeper/ZooCache.java

```diff
@@ -206,6 +209,7 @@ public ZooCache(ZooReader reader, Watcher watcher) {
     this.zReader = reader;
     nodeCache = new ConcurrentHashMap<>();
     this.externalWatcher = watcher;
+    log.trace("{} created new cache", cacheId, new Exception());
```

Comment:
Why do you want to add an exception here?

Reply:
That is so I could see a stack trace for what created the zoocache.  I was experimenting with this change and running some ITs, found that useful.